example/standard_schemas.js

```diff
@@ -1,51 +1,10 @@
 import Schema from '../src/modules/schema';
 import { validateCredentialSchema, expandJSONLD } from '../src/utils/vc';
 
-import bolSchema from '../src/utils/vc/schemas/bol';
```

Comment:
Why are some of the schemas being removed?

Reply:
the credentials are failing to be parsed as valid JSON-LD because it uses URLs that don't exist

Reply:
atleast in the 2.0.2 version these fail